integration-tests/src/tests/client/state_dump.rs

```diff
@@ -27,7 +27,7 @@ use std::sync::Arc;
 #[test]
 /// Produce several blocks, wait for the state dump thread to notice and
 /// write files to a temp dir.
-fn test_state_dump() {
+fn slow_test_state_dump() {
```

Comment:
Tbh it doesn't bother me. Usually finishes in ~4s or so, I ran it a few times and it didn't fail. I think normal CI has some retries as well, I remember seeing some "FLAKY" lines in the nextest output :thinking:

Reply:
@nagisa WDYT?

Reply:
Our CI has test retry mechanisms as well. The only reason to move it would be to move it out of codecov's sight as it can't handle tests SIGSEGVing. Simply ignoring this (or any) test segfaulting seems suspect to me.